perfkitbenchmarker/benchmarks/nginx_wrk_benchmark.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Run WRK against a simple nginx web server.
+
+`wrk` is a scalable web load generator.
+`nginx` is a popular static web server and proxy.
+"""
+
+import functools
+import logging
+import urlparse
+
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.packages import nginx
+from perfkitbenchmarker.packages import wrk
+
+
+flags.DEFINE_float('nginx_wrk_max_p50_ratio', 5.0,
+                   'Double concurrent connections until '
+                   'p50 / (1 connection p50) > nginx_wrk_max_p50_ratio')
+
+FLAGS = flags.FLAGS
+
+BENCHMARK_NAME = 'nginx_wrk'
+BENCHMARK_CONFIG = """
+nginx_wrk:
+  description: Run wrk against nginx.
+  vm_groups:
+    server:
+      vm_spec: *default_single_core
+    client:
+      vm_spec: *default_single_core
+"""
+
+
+def GetConfig(user_config):
+  return configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
+
+
+def _PrepareServer(vm):
+  """Installs nginx on the server."""
+  vm.Install('nginx')
+  vm.RemoteCommand('echo OK | sudo tee /usr/share/nginx/html/OK')
+  nginx.Start(vm)
+
+
+def _PrepareClient(vm):
+  """Install WRK on the client VM."""
```

Comment:
super nit: looks like the correct capitalization of wrk is all lower case (based on their github page)